datacenter/dtr/2.2/guides/admin/upgrade/release-notes.md

```diff
@@ -10,6 +10,27 @@ known issues for each DTR version.
 You can then use [the upgrade instructions](index.md),
 to upgrade your installation to the latest release.
 
+## DTR 2.2.3
+
+(27 Feb 2017)
+
+**New features**
+
+* The tags list API now supports pagination
+
+**General improvements**
+
+* Several improvements to `docker/dtr` to make it more resilient
```

Comment:
shouldn't this be "Improvements to the bootstrap to make it more resilient"?  I'm not sure most people will know what `docker/dtr` is.